app/(dashboard)/mailboxes/[mailbox_slug]/[category]/ticketCommandBar/suggestedActions.tsx

```
@@ -56,7 +56,7 @@ export function SuggestedActions({
                 </Button>
               );
             case "assign":
-              const assignee = orgMembers?.find((m) => m.id === t.clerkUserId);
+              const assignee = orgMembers?.find((m) => m.id === t.userId);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)